src/sandstorm/run-bundle.c++

```diff
@@ -1183,18 +1189,17 @@ private:
     KJ_SYSCALL(sigprocmask(SIG_SETMASK, &sigset, nullptr));
   }
 
-  void copyEtc() {
+  void linkEtc() {
     auto files = splitLines(readAll("etc.list"));
```

Comment:
As a side note: @zarvox you may know more about this than I do - is there a _directory_ of resolv.conf entries (`/etc/resolv.conf.d` ?) on some systems? If so we should think about adding them to `etc.list`.

Not necessarily on-topic for this pull request, but thought I'd mention.